sale_advance_payment/models/sale.py

```diff
@@ -86,7 +86,8 @@ def _compute_advance_payment(self):
             for inv in order.invoice_ids.filtered(
                 lambda x: x.move_type == "out_invoice"
             ):
-                invoice_paid_amount += inv.amount_total - inv.amount_residual
+                if inv.state == "posted":
```

Comment:
How about we just add this condition in `filtered` instead of using both `filtered` and `if`?

Reply:
@sistecem Minor enhancement